index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt ("ingrese su nombre") ;
+alert(`Hola ${nombreUsuario}`) ;
```

Comment:
Aca no deberia usar interpolacion de strings
```suggestion
alert("Hola" + nombreUsuario) ;
```